Commands.md

```diff
@@ -361,6 +362,17 @@ Does the following:
   - Executes _.git/hooks/post-release.sh_ (if present)
 
 
+## git rename-branch
+
+Rename a branch (locally and remotely).
```

Comment:
In my opinion, it is worth to clarify that this command doesn't really rename a remote branch. Seems it rename the upstream of local branch?